.release-notes/3738.md

```diff
@@ -0,0 +1,5 @@
+## Add IsPrime to math package
+
+Quickly determine if a given number is prime using the 6k ± 1 method.
+
+All integers (excluding 2 and 3), can be expressed as (6k + i), where i = −1, 0, 1, 2, 3, or 4. Since 2 divides (6k + 0), (6k + 2), and (6k + 4), while 3 divides (6k + 3) that leaves only (6k - 1) and (6k + 1) for expressing primes
```

Comment:
Nice. Given that these notes end up in markdown on GitHub. Feel free to do `IsPrime` and `math` is you want or to use other markdown formatting as well. (You don't have to though)

Reply:
Done